packages/core/src/application.ts

```diff
@@ -280,141 +32,89 @@ export class Application extends Context {
    * modifications.
    *
    * ```ts
-   * @spec(apiSpec)
    * class MyController {
    * }
    * app.controller(MyController).lock();
    * ```
    */
-  controller(controllerCtor: ControllerClass): Binding {
+  controller(controllerCtor: Constructor<{}>): Binding {
     return this.bind('controllers.' + controllerCtor.name).toClass(
       controllerCtor,
     );
   }
 
   /**
-   * Register a new Controller-based route.
+   * Bind a Server implementation to the Application's master context.
+   * Each server added in this way must provide a unique prefix
+   * to prevent binding overlap.
    *
    * ```ts
-   * class MyController {
-   *   greet(name: string) {
-   *     return `hello ${name}`;
-   *   }
-   * }
-   * app.route('get', '/greet', operationSpec, MyController, 'greet');
+   * app.server(new RestServer(app, options));
+   * // All of this server's bindings are under "servers.RestServer".
+   * app.server(new RestServer(app, options), "v1API");
+   * // All of this server's bindings are under "servers.v1API".
```

Comment:
Is this true? How are we achieving namespacing of per-server bindings in application context? I mean where is the code adding the namespace to every server-level binding?

Reply:
I think that was true in a previous iteration, I'll clean up the comment.